### PR TITLE
Make development section in README.md more explicit

### PR DESCRIPTION
**Pull request by sirhcel**
The initial startup for serving the web page locally takes quite some time. Let's take other young players by hand through this long wait.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,31 @@
...
+```
+🍾 Site built into ./target
+  129 files generated in 62.64 seconds
+
+  Server started at:
+  http://localhost:3000/ (local)
+  http://192.0.2.3:3000/ (network)
+```
+
+When doing heavy development, you might end in the need to provide a GitHub API
```
Comment (Yatekii) [inline README.md:43]:
```suggestion
When doing heavy development, you might end in the need to provide a GitHub API Token
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,31 @@
...
+(for fetching the probe-rs repository in the background). See
+[here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+for hot to get yours, save it to a file like `~/.probe-rs-github-token` and
```
Comment (Yatekii) [inline README.md:46]:
```suggestion
for how to get yours, save it to a file like `~/.probe-rs-github-token` and
```

File: README.md (hunk 1 above)
Comment (sirhcel) [reply README.md:43]: Thanks! Fixed.

File: README.md (hunk 2 above)
Comment (sirhcel) [reply README.md]: Thanks! Fixed.

**Review by Yatekii — APPROVED** [review]

**Pull request merged**